node/10/Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM node:10-slim
 MAINTAINER Damien Carcel <damien.carcel@akeneo.com>
 
+ARG USER_ID=1000
+
+# Delete and recreate node user with the provided id
+RUN userdel --remove node \
+    && groupadd -g $USER_ID node \
+    && useradd -lms /bin/bash -u $USER_ID node -g node
```

Comment:
Quite right - not sure why I'd done it that way, since I am using `usermod` and `groupmod` in the changes I'm working on for the other containers.
I'll fix that up.